README.md

```diff
@@ -184,18 +181,19 @@ Adding `bundle exec` ensures you're using the version of passenger you just inst
 
 ## Reply-by-email
 
-Public Lab now supports reply by email to comment feature. For more details regarding it go to the [email documentation](https://github.com/publiclab/plots2/blob/main/doc/EMAIL.md)
+Public Lab now supports reply by email to comment features. For more details, go to the [email documentation](https://github.com/publiclab/plots2/blob/main/doc/EMAIL.md)
```

Comment:
I believe we would like to keep it as singular because the "reply by email comment" is one feature..what do you think?

Reply:
Ohh, that makes a lot of sense. If that's the case wouldn't be better to string them together like "reply-by-email-to-comment" feature? Or put them in quotes so anyone reading knows that they go together

Reply:
I agree, the quotes would make the phrase better. Thanks

Reply:
How do I update the PR?

Reply:
Update the changes from here https://github.com/OlaoluwaM/plots2/tree/patch-1 and commit, the commit will update over here. thanks

Reply:
Done!